Gems/Atom/Asset/Shader/Code/Source/Editor/AzslShaderBuilderSystemComponent.cpp

```diff
@@ -83,7 +83,7 @@ namespace AZ
             // Register Shader Asset Builder
             AssetBuilderSDK::AssetBuilderDesc shaderAssetBuilderDescriptor;
             shaderAssetBuilderDescriptor.m_name = "Shader Asset Builder";
-            shaderAssetBuilderDescriptor.m_version = 121; // Add support for metal shader language 2.2
+            shaderAssetBuilderDescriptor.m_version = 122; // Updating dxc to version 1.7.2308
```

Comment:
You will need to bump line 111 as well
shaderVariantAssetBuilderDescriptor.m_version = 38; // Add support for metal shader language 2.2